src/pages/home/home.scss

```diff
@@ -25,3 +25,8 @@ page-home {
   }
 
 }
+
+ion-app.cameraView page-home {
+  display: none;
+}
+
```

Comment:
it can work like this also, or no ?
```
page-home {
  ion-app.cameraView {
    display: none;
  }
}
```

Reply:
no, that will translate in css into `page-home ion-app.cameraView` which is diff than `ion-app.cameraView page-home`

Reply:
ahh yup cool